pkg/hope/entities.go

```diff
@@ -129,6 +135,7 @@ type Node struct {
 	Network    string
 	Cpu        int
 	Memory     int
+	Taints     []NodeTaint
```

Comment:
_Technically_ the format should be compatible with `core/v1/Taint`, but I would feel kinda weird creating that dependency.